### PR TITLE
Update hkust_data_prep.sh

### PR DESCRIPTION
**Pull request by zh794390558**
add `set -e` for  `python -c "import mmseg" 2>/dev/null || \
  (echo "mmseg is not found. Checkout tools/extra/install_mmseg.sh" && exit 1;)` not exit as expected.

### CODE REVIEW

**Comment by kkm000** [comment]
Thanks. `set -e` in itself is not wrong.

As for the code,
https://github.com/kaldi-asr/kaldi/blob/952bfd84d4325a50f7c64e16df66519c16581ece/egs/hkust/s5/local/hkust_data_prep.sh#L68-L72

the `echo` and `exit` are run in a subshell. The `exit` exits the subshell with code 1 and returns back to the program, which continues to run happily This is the actual bug.. What was intended here was running the echo and exit in a *group command* instead of a subshell:
```sh
python -c "import mmseg" 2>/dev/null || {
    echo "mmseg is not found. Checkout tools/extra/install_mmseg.sh"; exit 1; }
```
If you are up to helping with a better fix, please fix this bug. Also, move the whole check to an earlier point, so the script fails quickly if the module not found. I'd put as early as before the first `mkdir` at the line 18. And even better, I'd reword the error message for clarity, "Python module mmseg is not found. To install it, run tools/extra/install_mmseg.sh".

**Comment by zh794390558** [comment]
thanks for you recommendation。

**Comment by zh794390558** [comment]
please check.

File: egs/hkust/s5/local/hkust_data_prep.sh [hunk 1]
```diff
@@ -14,6 +14,11 @@ hkust_text_dir=$2
 train_dir=data/local/train
 dev_dir=data/local/dev
 
+# transcripts normalization and segmentation
+# needs external tools
+python -c "import mmseg" 2>/dev/null || {
```
Comment (danpovey) [inline egs/hkust/s5/local/hkust_data_prep.sh]: I think invoking this as python2.7 would be better, since mmseg doesn't work on python3, IIRC.  Obviously we should make sure that the script that actually uses mmseg later on also invokes it that way.

Comment (zh794390558) [reply egs/hkust/s5/local/hkust_data_prep.sh]: Done.
